R/mig_resid.R

```diff
@@ -579,18 +584,20 @@ mig_resid_stock <- function(pop_m_mat,
     is.numeric(ages),
     is.numeric(ages_fertility)
   )
-  # Check in dimensions are ok
-  if(ncol(asfr_mat) == ncol(pop_f_mat)-1){
+  # Check in dimensions are ok - still working on this
+  if(ncol(asfr_mat) == ncol(pop_f_mat) -1 & nrow(sr_f_mat) == nrow(pop_f_mat) -1){
     print("matrix dimensions are correct")
+    else {
+    print("check matrix dimension")
```

Comment:
elsewhere for console messages we've used `cat()` (making use of line breaks `\n` as needed. Functions have an argument called `verbose` (logical), which also toggles whether to send these messages (usually an `if` inside of an `if`).